### ISSUE TITLE
Jicofo stopped working after removing the component code

### ISSUE THREAD

**Issue by licaon-kter** [jitsi/jicofo#680]
Debian Buster, arm64, self compiled

Albeit https://github.com/jitsi/jicofo/pull/672 removes the need for component subdomain, port and secret, 95d2b77d185145d54f72df6e5a54ce2affff7bdf fails for me with:
```
SEVERE: [1] org.jitsi.jicofo.Main.log() An uncaught exception occurred in thread=Thread[main,5,main]
org.jitsi.cmd.ParseException: Some of required arguments were not specified: [secret]
        at org.jitsi.cmd.CmdLine.parse(CmdLine.java:132)
        at org.jitsi.jicofo.Main.setSystemProperties(Main.java:136)
        at org.jitsi.jicofo.Main.main(Main.java:58)
```
...so I'm forced to setup a secret since the code demands it:
https://github.com/jitsi/jicofo/blob/3936f654fc90d0aac9c109efc50d692ab3f457d6/src/main/java/org/jitsi/jicofo/Main.java#L130

Now, with that secret _(or a random string)_, component presence does not matter _(tried with and now without)_, it still won't connect, because **it tries to connect to the component** _(that you've just removed)_:
```
[conference.js] <_onConferenceFailed>:  CONFERENCE FAILED: conference.focusDisconnected focus.meet.domain.tld 8 app.bundle.min.js:106:2382
    o app.bundle.min.js:106
    _onConferenceFailed app.bundle.min.js:122
    emit events.js:152
    emit events.js:152
    _allocateConferenceFocusError moderator.js:375
    allocateConferenceFocus moderator.js:291
    a strophe.umd.js:2716
    run strophe.umd.js:1875
    _dataRecv strophe.umd.js:3157
    forEachChild strophe.umd.js:830
    _dataRecv strophe.umd.js:3146
    _onRequestStateChange strophe.umd.js:5012
    _newXHR strophe.umd.js:4452
    Request strophe.umd.js:4382
    _onIdle strophe.umd.js:4899
    _onIdle strophe.umd.js:3881
    flush strophe.umd.js:2599
    flush XmppConnection.js:340
    allocateConferenceFocus moderator.js:297
    allocateConferenceFocus moderator.js:283
    _allocateConferenceFocusError moderator.js:384
    (Async: setTimeout handler)
    _allocateConferenceFocusError moderator.js:383
    allocateConferenceFocus moderator.js:291
    a strophe.umd.js:2716
    run strophe.umd.js:1875
    _dataRecv strophe.umd.js:3157
    forEachChild strophe.umd.js:830
    _dataRecv strophe.umd.js:3146
    _onRequestStateChange strophe.umd.js:5012
    _newXHR strophe.umd.js:4452
    Request strophe.umd.js:4382
    _onIdle strophe.umd.js:4899
    _onIdle strophe.umd.js:3881
    flush strophe.umd.js:2599
    flush XmppConnection.js:340
    allocateConferenceFocus moderator.js:297
    allocateConferenceFocus moderator.js:283
    _allocateConferenceFocusError moderator.js:384
    (Async: setTimeout handler)
    _allocateConferenceFocusError moderator.js:383
    allocateConferenceFocus moderator.js:291
    a strophe.umd.js:2716
    run strophe.umd.js:1875
    _dataRecv strophe.umd.js:3157
    forEachChild strophe.umd.js:830
    _dataRecv strophe.umd.js:3146
    _onRequestStateChange strophe.umd.js:5012
```
...this on https://github.com/jitsi/jitsi-meet/commit/fe78f104bc304beae342c0971854793402dde401 and https://github.com/jitsi/jitsi-videobridge/commit/27bafcb1075e8a44925d88ebe0b43a11365a7581

I also removed the component line in config.js _(and elsewhere where I could find it)_ so I guess it just composes "focus+domain" and tries that as the component,

**Comment by damencho** [comment]
You should not touch config.js
All you need to do is:
https://github.com/jitsi/jitsi-meet/blob/a39905883fffb967f11977edac860784fb127427/debian/jitsi-meet-prosody.postinst#L160
https://github.com/jitsi/jitsi-meet/blob/a39905883fffb967f11977edac860784fb127427/debian/jitsi-meet-prosody.postinst#L153
https://github.com/jitsi/jitsi-meet/blob/a39905883fffb967f11977edac860784fb127427/debian/jitsi-meet-prosody.postinst#L165

This is automatically taken care of when updating the Debian packages.

**Comment by damencho** [comment]
We will be dropping the unneeded params.

**Comment by bgrozev** [comment]
> I also removed the component line in config.js

You shouldn't do that

**Comment by licaon-kter** [comment]
@bgrozev 
> You shouldn't do that

I removed it after trying with it, no change.

@damencho 
> All you need to do is:

Right, forgot to say, it's ejabberd, will need to translate those.

> This is automatically taken care of when updating the Debian packages.

I build from git the Debian packages, and I saw that client proxy line, and removed it because **after a PR that removes the focus component usage, it sounds wrong to have... focus components settings**.

Will try to take another look given this new information.
